src/kubectl.ts

```diff
@@ -219,7 +219,7 @@ async function asLines(context: Context, command: string): Promise<Errorable<str
     const shellResult = await invokeAsync(context, command);
     if (shellResult.code === 0) {
         let lines = shellResult.stdout.split('\n');
-        lines.shift();
+        // lines.shift();
```

Comment:
Won't this break other consumers of `asLines` who expect the headers to have been stripped off?